NEW.md

```diff
@@ -0,0 +1,261 @@
+# Stitch (for Atom / Hydrogen)
```

Comment:
Thanks this is useful for the review. I'll read in more detail later, but eventually this file should be removed and relevant bits added to the docs.